src/components/GameState.js

```diff
@@ -2,22 +2,24 @@ import React, { useState } from "react";
 import PropTypes from "prop-types";
 
 const GameState = ({ total, children }) => {
-	const [items, setItems] = useState(total);
-	const onCorrect = () => setItems(items - 1);
-	const onIncorrect = () => setItems(items + 1);
-	const onReset = () => setItems(total);
+    const [items, setItems] = useState(total);
+    const onCorrect = () => setItems(items - 1);
+    const onIncorrect = () => setItems(items + 1);
+    const onReset = () => setItems(total);
 
-	return (
-		<>
-			{ items === 0 ? <p className="text-white bg-dark text-center">Game Completed!</p> : null }
-			{children({ onCorrect, onIncorrect, onReset })}
-		</>
-	);
+    return (
+        <>
+            {items === 0 ? (
+                <p className="text-white bg-dark text-center">Game Completed!</p>
+            ) : null}
```

Comment:
since you're returning something only if _items === 0_ I'd simplify it to: 
`{items === 0 && <p className="text-white bg-dark text-center">Game Completed!</p>}`

Reply:
feel free to fix - not really related to this PR tho
We'll be fixing these types of 'false positive diffs' shortly, see #52